rcl/include/rcl/timer.h

```diff
@@ -32,14 +32,15 @@ extern "C"
 #include "rcl/types.h"
 #include "rmw/rmw.h"
 
-struct rcl_clock_t;
-struct rcl_timer_impl_t;
+struct rcl_clock_s;
+struct rcl_timer_impl_s;
+typedef struct rcl_timer_impl_s rcl_timer_impl_t;
```

Comment:
Same question about removing the typedef.

Reply:
Same as above.